src/prefect/new_flow_engine.py

```diff
@@ -355,11 +355,32 @@ def get_hooks(self, state: State, as_async: bool = False) -> Iterable[Callable]:
         if not flow_run:
             raise ValueError("Task run is not set")
 
+        enable_cancellation_and_crashed_hooks = (
+            os.environ.get(
+                "PREFECT__ENABLE_CANCELLATION_AND_CRASHED_HOOKS", "true"
+            ).lower()
+            == "true"
```

Comment:
```suggestion
            in ["true", "1"]
```

Reply:
Only the `Runner` sets this environment variable, so we can keep the check narrow.